### PR TITLE
[Frontend][ONNX] Add LinearRegressor operator support

### PR DESCRIPTION
**Pull request by zxy844288792**
Add LinearRegressor operator support in relay onnx frontend.

### CODE REVIEW

File: python/tvm/relay/frontend/onnx.py [hunk 1]
```diff
@@ -3444,6 +3444,35 @@ def body_fn(*loop_inputs):
         return outputs
 
 
+class LinearRegressor(OnnxOpConverter):
+    """Operator converter for LinearRegressor."""
+
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
```
Comment (huajsj) [inline python/tvm/relay/frontend/onnx.py:3451]: document for parameter/return/function brief

**Review by comaniac — APPROVED** [review]
LGTM

File: python/tvm/relay/frontend/onnx.py (hunk 1 above)
Comment (comaniac) [reply python/tvm/relay/frontend/onnx.py:3451]: IMHO, given that all functions in this frontend do not have docstring, it should be fine for this PR to follow this convention for now.

**Comment by comaniac** [comment]
Thanks @zxy844288792 @huajsj

**Pull request merged**